src/features/wallet/chains/sepolia.ts

```diff
@@ -0,0 +1,19 @@
+import { Network } from '../models/network/types/Network';
+
+export const SepoliaChain: Network = {
+  chainId: 11155111,
+  chainName: 'Sepolia',
+  nativeCurrency: {
+    name: 'Sepolia ETH',
+    symbol: 'ETH',
+    decimals: 18,
+  },
+  rpcUrls: ['https://ethereum-sepolia-rpc.publicnode.com'],
+  blockExplorerUrls: ['https://sepolia.etherscan.io/'],
+  addressExplorerUrl: 'address',
+  transactionExplorerUrl: 'tx',
+  multicallAddress: '',
+  isTestChain: true,
+  isLocalChain: true,
```

Comment:
Could you set `isLocalChain=false`
Context: isLocalChain exist for identifying networks working only in local computer like Hardhat, Ganache etc.

Reply:
Ganache wouldn't work locally.  Changed for Sepolia.